fbfmaproom/fbfmaproom.py

```diff
@@ -406,7 +406,8 @@ def subquery_unique(base_query, key, field):
     with psycopg2.connect(**CONFIG["db"]) as conn:
         df = pd.read_sql(query, conn, params={"key": key})
     if len(df) == 0:
-        raise InvalidRequestError(f"invalid region {region_key}")
+        raise Exception
```

Comment:
Why does this work if you have two adjacent `raise`s?

Reply:
Fixed. It doesn't actually matter what exception is raised here, but I removed the less specific one.

Reply:
@kgraaf please approve.